apps/web/src/seo/SeoDescriptions.ts

```diff
@@ -0,0 +1,212 @@
+import { TbLayoutAlignMiddle } from "react-icons/tb";
+
+type SeoProps = {
+  metaDescription: string;
+  metaTitle: string;
+};
+
+type Layouts = "Marketing" | "Blog" | "Integrations" | "DashBoard" | "Docs";
+
+export function getSeo(pageName: string, layoutName: Layouts) {
+  switch (layoutName) {
+    case "Marketing":
+      return getMarketingSeo(pageName);
+    case "Blog":
+      return getBlogLayout(pageName);
+    default:
+      return getMarketingSeo(pageName);
+  }
+}
+
+export function getMarketingSeo(pageName: string): SeoProps {
+  console.log("pageName", pageName);
+  const normalizedKey = pageName.startsWith("/") ? pageName.slice(1) : pageName;
+  switch (normalizedKey) {
+    case "":
```

Comment:
I honestly think having / in there makes it much more readable. `case ""` just looks a bit odd to me